services/plan_manager.js

```diff
@@ -80,3 +81,21 @@ exports.process_update_plan_input_req = async (input_data) => {
 exports.post_process_plan_batch = async (resp) => {
     resp.status(200).send({status:"success",message:"plan updated successfully"})
 }  
+
+exports.pre_process_get_all_plans = async (req) => {
+    const allPlans = await models.SubscriptionPlan.findAll()
+    if (allPlans) {
+        return allPlans;
+    } else {
+        throw new Api500Error(`Bad Request`)
```

Comment:
500 Error code is internal server error . 400 error code is bad request.Also remove the if else condition .exception case is handled in catch of controller.so just  return allPlans.